src/controllers/userController.js

```diff
@@ -100,4 +100,54 @@ const logOut = (req, res) => {
   }
 };
 
-module.exports = { signUp, logIn, logOut };
+/******************************************************
+ * @UPDATEUSER
+ * @route /api/v1/updateuser
+ * @method PUT
+ * @description singUp function for creating new user
+ * @body name, email, password, confirmPassword
+ * @returns User Object
+ ******************************************************/
+
+const updateUser = async (req, res) => {
+  try {
+    const { userId } = req.user; //retrieve user.id from jwtverify()
```

Comment:
Could you clarify where `jwtverify()` comes from?

Reply:
Also, it's not a part of `req.body`. Could you explain how you get access to `req.user`?